arrow-core/src/main/kotlin/arrow/core/extensions/setk.kt

```diff
@@ -23,18 +22,27 @@ import arrow.typeclasses.Show
 import arrow.typeclasses.hashWithSalt
 import arrow.core.combineK as setCombineK
 
-@extension
+@Deprecated(
+  "Typeclass instance have been moved to the companion object of the typeclass.",
+  ReplaceWith("Semigroup.setK<A>()", "arrow.core.extensions", "arrow.core.extensions"),
```

Comment:
```suggestion
  ReplaceWith("Semigroup.set<A>()", "arrow.core.set", "arrow.typeclasses.Semigroup"),
```

You need to use fully qualified packages in the `ReplaceWith` statement. `SetK` is getting removed so we're not going to add an extension on `Semigroup` for it.